.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  extends: [require.resolve('eslint-config-fusion')],
+  rules: {
+    'no-unused-vars': [
+      'error',
+      {vars: 'all', args: 'none', varsIgnorePattern: '^h$'},
```

Comment:
Is this mean to be here? I assume this has to do with JSX?

Reply:
I think it might be an old config. It seems to be a bad copy-paste from some preact thing